angr/sim_type.py

```diff
@@ -252,30 +255,36 @@ class SimTypeLongLong(SimTypeInt):
 class SimTypeChar(SimTypeReg):
     """
     SimTypeChar is a type that specifies a character;
-    this could be represented by an 8-bit int, but this is meant to be interpreted as a character.
+    this could be represented by a byte, but this is meant to be interpreted as a character.
```

Comment:
"8-bit int" is meant to indicate something more specific than "a byte" - it means that this class adds a particular semantic advantage over just using SimTypeInt(8) or whatever. you can change this to "a byte-width int"